### ISSUE TITLE
[SUGGESTION] Chams Improvements

### ISSUE THREAD

**Issue by jumboman32** [pyroclient/pyro#383]
I suggest wireframe, skeleton, and full overlay options for chams so that it can be more customizable.

ex.
https://gyazo.com/e4c83c4b8712cea5e98110019d65632b
https://gyazo.com/5596a882a2c1f49e575e5a63f7b4c3d1

**Comment by 3S0** [comment]
Agreed. The ESP is very limited in customizability and this would be very good to see.

**Comment by u3o** [comment]
People will hate pyro even more after that because it is taken idea from rhack

**Comment by 3S0** [comment]
> People will hate pyro even more after that because it is taken idea from rhack

It is a visual feature. rhack isn't the only client with wireframe.

**Comment by u3o** [comment]
> > People will hate pyro even more after that because it is taken idea from rhack
> 
> It is a visual feature. rhack isn't the only client with wireframe.

Then ok
